languages/en.ini

```diff
@@ -410,8 +410,25 @@ export_selected_favorites = "Export Selected Items"
 export_send = "Send to %%service%%"
 export_success = "Export Ready"
 export_unsupported_format = "Unsupported Export Format"
+external_auth_access_heading = "Request for Access"
+external_auth_access_login_message = "An external application is requesting access to your data. Please log in to review and authorize the request."
+external_auth_allow_access = "Allow Access"
+external_auth_deny_access = "Deny Access"
 external_auth_heading = "Access to licensed material"
+external_auth_ils_prompt = "Some of the requested information is retrieved from your library catalog profile."
 external_auth_login_message = "Login to access licensed material"
+external_auth_prompt_html = "is requesting the following access rights:"
```

Comment:
This seems to be missing the client name token.

Reply:
The token was a leftover. The client name is in the heading.